tracer/src/Datadog.Trace/Ci/CIVisibility.cs

```diff
@@ -146,7 +151,7 @@ internal static void Flush()
             try
             {
                 SynchronizationContext.SetSynchronizationContext(null);
-                AsyncUtil.RunSync(() => FlushAsync(), cts.Token);
+                AsyncUtil.RunSync(FlushAsync, cts.Token);
```

Comment:
OMG, did we finally get method group caching in C#11?!

![image](https://media.giphy.com/media/5GoVLqeAOo6PK/giphy.gif)